selfdrive/car/hyundai/values.py

```diff
@@ -2104,6 +2106,14 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x00GL3_ RDR -----      1.00 1.02 99110-L8000         ',
     ],
   },
+  CAR.KIA_SPORTAGE_PHEV_5TH_GEN: {
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00NQ5__               1.01 1.03 99110-CH000         ',
+    ],
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00NQ5 FR_CMR AT USA LHD 1.00 1.01 99211-P1060 680',
+    ],
+  },
```

Comment:
Thanks @sunnyhaibin for help with the car port for my 2023 Kia Sportage PHEV X-Line Prestige.

Switching conversion here from [discord](https://discord.com/channels/469524606043160576/1167525974896803870/1180365497947148380):

> install with the following URL: `installer.comma.ai/sunnyhaibin/kia-sportage-phev-2023-port`. Once done, gather a route, and upload all rlogs and preserve the route per the instructions.

Here is a route preserved and shared on [comma connect](https://connect.comma.ai/428412ad82894ba3/1701527340790/1701527626826) with ID `428412ad82894ba3|2023-12-02--09-29-01--0`.

Also noting that I get a banner at the bottom of the screen with "CAN Error: Check Connections". I restarted the car a few times and this error message remained. Did not notice any loose or misconfigured cables.